src/iap.ts

```diff
@@ -29,65 +29,43 @@ const ANDROID_ITEM_TYPE_SUBSCRIPTION = 'subs';
 const ANDROID_ITEM_TYPE_IAP = 'inapp';
 
 let iapInstallSourceAndroid = InstallSourceAndroid.NOT_SET;
-let iapFallbackInstallSourceAndroid = InstallSourceAndroid.GOOGLE_PLAY;
-
-export function setFallbackInstallSourceAndroid(
-  installSourceAndroid: InstallSourceAndroid,
-): void {
-  iapFallbackInstallSourceAndroid = installSourceAndroid;
-}
 
+/**
+ * This function will be made internal (not exported) in future versions.
```

Comment:
Could you kindly point to the related doc for this? The `deprecation` notice.

Reply:
https://github.com/dooboolab/react-native-iap/pull/1374/files#diff-d710196e59f78f9faa00b02fa321ef3cf62914e393feecb47a9b6d6f1f8215b4R32

Reply:
Ok looks like this `deprecation` notice will be handled by some other experts like you :laughing: